docker.cabal

```diff
@@ -1,5 +1,5 @@
 name:                docker
-version:             0.6.0.6
+version:             0.6.0.7
```

Comment:
Wouldn't this warrant a major version bump?

Reply:
I don't *think* so, unless you are considering the changes in the other PRs/recent changes as well. The change in #79 is probably a breaking change so that may justify that the next version should be `0.7`. The change in #92 is a minor change.

For this PR: nothing that worked previously should be affected by these changes and we haven't added any new exports/haven't changed any types of the exported definitions in `Docker.Client.Types`.

In any case, that's probably moot if the next release is 0.7 anyway. I will rebase this PR on `master` and bump the version to 0.7.

Reply:
Yeah I think you're right...I posted this and only after realized the compatibility functions and CPP's below. 

In any case...you're also right that there's a bunch of other changes already in master that probably warrant the major version bump anyway.